### ISSUE TITLE
Add Scroll to Berachain

### ISSUE THREAD

**Issue by hvregela** [scroll-tech/contribute-to-scroll#1474]
### Name of the project

Berachain

### Project type

Tool

### Site of the project

http://berachain.com/

### Github repository of the project

https://github.com/berachain

### Networks

Scroll Testnet - Sepolia

### Additional context

I would like to see Scroll integrated with the Bearachain Network on short notice for bringing capabilities to and from Berachain and I would like to see Scroll integrated with the Berachain features  in order to facilitate this

**Comment by Bykibayo** [comment]
yes Berachain while still in its formation stage has alot of people interested in it.
